docs/guide/writing/docs-principles.rst

```diff
@@ -114,22 +44,32 @@ Participatory
 users.*
 
 Integrate documentation into the standard workflow of developers, and
-seek to reduce silos that solicit documentation from only a subset of
+reduce silos that solicit documentation from only a subset of
 the software's contributors. Developers and engineers are the people
 with the best access to in-demand information, and getting them to
 document it will help foster a *culture* of documentation.
 
-As well, documentation *readers* (i.e., users) should have clear avenues
+Documentation *readers* (i.e., users) should also have clear avenues
 towards involvement in documentation. A good first step is to give
-readers the ability to offer feedback in the form of comments or
+readers the ability to offer feedback through comments or
 suggestions. Allowing readers to edit documentation directly (e.g., in a
 wiki) can also be effective but must be weighed against the need and
 capacity for editorial oversight.
 
 Encourage *everyone* to become a :doc:`documentarian </documentarians>`!
 
-Content should be...
---------------------
+Principles for great content
+-----------------------------
+
+*"Content" is the conceptual information within documentation.*
```

Comment:
Do we have a source for this? I'm not sure it's a thing

Reply:
🤷